custom/ewsghana/templates/ewsghana/partials/location_async.html

```diff
@@ -0,0 +1,32 @@
+{% load hq_shared_tags %}
```

Comment:
i see. can you just add a section to the base template and then override it here?